src/sage/combinat/symmetric_group_algebra.py

```diff
@@ -1586,6 +1586,27 @@ def specht_module(self, D):
         from sage.combinat.specht_module import SpechtModule
         return SpechtModule(self, D)
 
+    def tabloid_module(self, D):
+        r"""
+        Return the module of tabloids with the natural action of ``self``.
+
+        .. SEEALSO::
+
+            :class:`~sage.combinat.specht_module.TabloidModule`
+
+        EXAMPLES::
+
+            sage: SGA = SymmetricGroupAlgebra(QQ, 5)
+            sage: TM = SGA.tabloid_module(Partition([3,1,1]))
+            sage: TM
+            Tabloid module of [3, 1, 1] over Rational Field
+            sage: s = SymmetricFunctions(QQ).s()
```

Comment:
really need the conversion to Schur here ?

Reply:
Relic of the previous commits. Removes.